fbchat/client.py

```diff
@@ -137,11 +139,11 @@ def _graphql(self, payload, error_retries=3):
             raise e
 
     def _cleanGet(self, url, query=None, timeout=30):
-        return self._session.get(url, headers=self._header, params=query, timeout=timeout)
+        return self._session.get(url, headers=self._header, params=encode_params(query), timeout=timeout, verify=self.ssl_verify)
```

Comment:
You forgot to add the `verify` in `_postFile`

Reply:
Done